template/db.py

```diff
@@ -18,18 +19,28 @@ def close(self):
     :param num_columns: int     #Number of Columns: all columns are integer
     :param key: int             #Index of table key in columns
     """
+
     def create_table(self, name, num_columns, key):
         table = Table(name, num_columns, key)
+        self.tables.append(table)
         return table
 
     """
     # Deletes the specified table
     """
+
     def drop_table(self, name):
-        pass
+        for table in self.tables:
+            if table.name == name:
+                self.tables.remove(table)
 
     """
     # Returns table with the passed name
     """
+
     def get_table(self, name):
-        pass
+        for table in self.tables:
+            if table.name == name:
+                return table
+
+        raise Exception("table: " + name + " not found in database")
```

Comment:
maybe instead of raising exection, put it in a try except block, so that it doesn't crash on an invalid name, instead just return False or something